src/main/webapp/electionInfo.html

```diff
@@ -0,0 +1,83 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <title>gVote</title>
+
+    <link href="https://fonts.googleapis.com/css2?family=Nunito+Sans:ital,wght@0,400;0,700;1,400&display=swap" rel="stylesheet">
+
+    <!--Bootstrap Libraries-->
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.4.1/css/bootstrap.min.css">
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.5.1/jquery.min.js"></script>
+    <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.4.1/js/bootstrap.min.js"></script>
+    <script src="https://cdn.jsdelivr.net/npm/handlebars@latest/dist/handlebars.js"></script>
+
+    <script src="https://maps.googleapis.com/maps/api/js?key=AIzaSyBlNN_uUrbqY2uRaBpKTD-5Jbw6nk8n0_k&libraries=places&callback=initAutocomplete" async defer></script>
```

Comment:
Is there some more setup we need to do for the api key? I got an ApiNotActivatedMapError when trying to run this locally on my machine.
![Screen Shot 2020-07-28 at 5 09 31 PM](https://user-images.githubusercontent.com/3631730/88741765-592e6880-d0f5-11ea-800b-0fad3a12fc07.png)